src/Image.h

```diff
@@ -25,5 +25,6 @@ namespace cs
         static Image fromPng(Stream& stream);
         void toPng(Stream& stream) const;
         Image crop(int32_t cropX, int32_t cropY, uint32_t cropWidth, uint32_t cropHeight) const;
+        Image copy();
```

Comment:
Perhaps use the copy constructor instead?

Reply:
I wanted to avoid creating all the others, as per rule of 5. Would you rather have all of them or this method?

Reply:
What is rule 5?

You should disable the copy constructor if it isn't implemented, because it will do a shallow copy which is probably not a good idea.

Reply:
Rule of 5 means if you create one constructor, you need to create all 5 - if none are defined, they are auto-generated, but not if any is present.

Reply:
You also cannot copy currently, as unique_ptr member field has copying disabled, so no need to disable it currently by us